<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QListView" name="appsView"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="buttonsLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="setAsDefault">
       <property name="text">
        <string>Set / clear default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addApplication">
       <property name="text">
        <string>Add application...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toggleApplication">
       <property name="text">
        <string>Remove application</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
